### PR TITLE
Add Cpp file test

### PR DESCRIPTION
**Pull request by Mithil467**
Add C++ file to collection of syntax highlighting text files #1213

### CODE REVIEW

File: tests/syntax-tests/source/Cpp/test.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
+#include <bits/stdc++.h> // includes most of the useful libraries
+#include "mainwindow.h"
+
+using namespace std;
+
+/* This C program was submiited to help bat
```
Comment (sharkdp) [inline tests/syntax-tests/source/Cpp/test.cpp:6]: typo

File: tests/syntax-tests/source/Cpp/test.cpp [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            pq.pop();
+        }
+
+        for (int i = 0; i < n; i++)
+            cout << a[i] << " "; // print A
+
+        cout << endl;
+    }
+    return 0;
+}
```
Comment (sharkdp) [inline tests/syntax-tests/source/Cpp/test.cpp:79]: Could you please add a trailing newline to the file? (See #975)

File: tests/syntax-tests/source/Cpp/test.cpp (hunk 1 above)
Comment (Mithil467) [reply tests/syntax-tests/source/Cpp/test.cpp:6]: Oops!

File: tests/syntax-tests/source/Cpp/test.cpp (hunk 2 above)
Comment (Mithil467) [reply tests/syntax-tests/source/Cpp/test.cpp]: Sure! Done :)

Comment (Mithil467) [reply tests/syntax-tests/source/Cpp/test.cpp]: Can you please confirm if it is fixed?

**Comment by sharkdp** [comment]
Thank you!

**Pull request merged**